src/shred_collector/shred_receiver.zig

```diff
@@ -106,12 +116,16 @@ pub const ShredReceiver = struct {
         comptime is_repair: bool,
     ) !void {
         if (packet.size == REPAIR_RESPONSE_SERIALIZED_PING_BYTES) {
-            if (try self.handlePing(&packet)) |p| try response_sender.send(p);
+            if (try self.handlePing(&packet)) |p| {
```

Comment:
rename `p` to `pong_packet` ?

Reply:
e6312ca0a63ec918fb67d3a891fcaab44e9464be